dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpBaseMessageImpl.java

```diff
@@ -2970,6 +2969,20 @@ else if (getServiceContext().getHttpConfig().onlySameSiteStar()) {
                 }
             }
 
+            // Must be in beta to check for SameSite=None Incompatible clients
+            if (ProductInfo.getBetaEdition() && cookie.getAttribute("samesite") != null && cookie.getAttribute("samesite").equals(HttpConfigConstants.SameSite.NONE.getName())) {
+                String userAgent = getServiceContext().getRequest().getHeader(HttpHeaderKeys.HDR_USER_AGENT).asString();
+                if (userAgent != null && SameSiteCookieUtils.isSameSiteNoneIncompatible(userAgent)) {
+                    //TODO: do we remove Secure, probably should be retained.
+                    Tr.debug(tc, "Incompatible client for SameSite=None found with the following User-Agent: " + userAgent);
```

Comment:
Tr.debug statements are more performant when wrapped around the: 

`if(TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()`

Reply:
Thanks for spotting that -- I created a PR here: https://github.com/OpenLiberty/open-liberty/pull/29506